docs/operators/field_caching.md

```diff
@@ -0,0 +1,158 @@
+### Field level caching configuration
+
+Field level caching enables caching for any field or query
+which contains a resolver.
+
+```graphql
+type Query {
+    posts: [Post] @http(path: "/posts")
+}
+
+type Post {
+    id: Int
+    title: String
+    user_id: Int @cache(maxAge: 100)
+    user: User @http(path: "/user/{{user_id}}") @cache(maxAge: 200)
+}
+
+type User {
+    id: Int
+    name: String
+    email: String
+}
+```
+
+For the above config, the result of `user` field will be cached because it contains an http resolver. However, the output of `user_id` will not be cached because it doesn't have a resolver for itself, it's value is fetched by the resolver at the `posts` query.
```

Comment:
```suggestion
For the above config, the result of `user` field will be cached because it contains an http resolver. However, the output of `userId` & `title` will not be cached because it doesn't have a resolver for itself, it's value is fetched by the resolver at the `posts` field having the `@http(path: "/posts")` directive.
```